cli/cmd/lql_delete.go

```diff
@@ -0,0 +1,75 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	lqlDeleteBadInputMsg string = "Please specify a valid query ID."
+	lqlDeleteDebugMsg    string = "deleting LQL query"
+	lqlDeleteSuccessMsg  string = "LQL query (%s) deleted successfully.\n"
+	lqlDeleteUnableMsg   string = "unable to delete LQL query"
```

Comment:
Same here, my opinion would be not do define constants for messages, but happy to hear the why behind this. 😉

Reply:
Partially discussed above.  Maybe this is more common in python where you would just set up a class of constant variables like this for output/logging messages.

1. This allows these to easily be referred to in test code if needed.
2. Allows for modification w/o actually touching function code.
3. Makes for easy templating of like functions

Reply:
@afiune and I discussed the merits of this approach and we're sorta on the same page that there are pro's and con's to doing it like this vs. not.  He was reasonable enough to allow this approach to be pursued for now and potentially changed in the future.